sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -302,8 +299,9 @@ public void run() {
         } catch (Exception e) {
           logger.log(Level.WARNING, "Exporter threw an Exception", e);
         }
+        return exportSize;
       } else {
-        logger.log(Level.FINE, "Exporter busy. Dropping spans.");
+        return 0;
```

Comment:
Returning zero is key - this is what'll cause the internal buffer to mount up and for the existing span-drop functionality of logging to kick in.